IssueCreator/IssueCreator/Main.cs

```diff
@@ -0,0 +1,372 @@
+﻿using IssueCreator.Models;
+using IssueCreator.Controls;
+using IssueCreator.Dialogs;
+using Octokit;
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Deployment.Application;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace IssueCreator
+{
+    public partial class frmMain : Form
+    {
+        private static Settings s_settings;
+        private static IssueManager s_issueManager;
+        private static IssueToCreate s_previouslyCreatedIssue;
+
+        private static string SettingsFile = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData), "issueCreator.settings");
+
+        public frmMain()
+        {
+            InitializeComponent();
+        }
+
+        private async void FrmMain_Load(object sender, EventArgs e)
+        {
+            s_settings = Settings.Deserialize(SettingsFile);
+
+            s_issueManager = IssueManager.Create(s_settings);
+
+            // if we don't have a github token, prompt settings.
+            if (s_issueManager == null)
+            {
+                ShowPreferencesDialog();
```

Comment:
One annoying thing here is you don't have anything in the Task Bar yet so tabbing around to this window after getting your token from the browser is kind of a pain.

Reply:
No idea how to fix this.